test-executor/src/main/java/com/cognifide/aet/executor/http/RerunDataWrapper.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * AET
+ *
+ * Copyright (C) 2013 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.cognifide.aet.executor.http;
+
+import com.google.common.base.Objects;
+
+public class RerunDataWrapper {
+
+    private String correlationId;
+    private String company;
+    private String project;
+    private String suite;
+    private String testUrl;
+    private String testName;
+
+    public RerunDataWrapper(String correlationId, String company, String project, String suite, String testUrl, String testName) {
```

Comment:
This constructor is probably redundant, I don't think that GSON is using it. Setters are also redudant here